src/event-helper.js

```diff
@@ -154,14 +155,13 @@ export function loadPromise(eleOrWindow) {
   const loadingPromise = new Promise((resolve, reject) => {
     // Listen once since IE 5/6/7 fire the onload event continuously for
     // animated GIFs.
-    const {tagName} = eleOrWindow;
-    if (tagName === 'AUDIO' || tagName === 'VIDEO') {
-      unlistenLoad = listenOnce(eleOrWindow, 'loadstart', resolve);
+    if (eleOrWindow instanceof HTMLMediaElement) {
```

Comment:
What elements are `HTMLMediaElement`?

Reply:
`audio` and `video`, I thought this would be a no-op. I can revert, and use this same test in `isLoaded`

Reply:
Done